guides-and-tutorials/sbtc/README.md

```diff
@@ -0,0 +1 @@
+The guides in this section provide step-by-step instructions for interacting with sBTC, including operating as a signer and (coming soon) developer guides on how to interact with sBTC as an application developer.
```

Comment:
We might want to mention that as of today you can only run an sBTC signer if you have been selected from SIP-28. I imagine as a user I could be confused trying to run one not knowing it is a permisonned set for the time being.